pvnet/models/base_model.py

```diff
@@ -390,26 +432,26 @@ def save_pretrained(
             link = f"https://wandb.ai/{wandb_repo}/runs/{wandb_id}"
             wandb_links += f" - [{link}]({link})\n"
 
+        # Find package versions for OCF packages
+        packages_to_display = ["pvnet", "ocf-data-sampler"]
+        packages_and_versions = {
+            dist.project_name: dist.version
+            for dist in pkg_resources.working_set
+            if dist.project_name.lower() in packages_to_display
+        }
```

Comment:
A bit of a nitpick but I feel that it would be cleaner to just ask for the versions of the two packages we are interested in instead of cycling through the entire list, so something like this:
```suggestion
        packages_and_versions = {
            package_name: pkg_resources.get_distribution(package_name).version
            for package_name in packages_to_display
        }
```

Reply:
> A bit of a nitpick but I feel that it would be cleaner to just ask for the versions of the two packages we are interested in instead of cycling through the entire list, so something like this:
> ```suggestion
>         packages_and_versions = {
>             package_name: pkg_resources.get_distribution(package_name).version
>             for package_name in packages_to_display
>         }
> ```

Agreed, thanks for the feedback. I'll sort this in the next few hours once I'm back at a laptop

Reply:
All sorted - have removed the case ambiguity also as the tests should now cover this.